### PR TITLE
FED-EXT:  Tujibebe review + cleanup

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

File: articles/templates/articles/_patterns/article-teasers-recommendations.html [hunk 1]
```diff
@@ -20,9 +20,22 @@ <h4 class="heading heading--related-up-next">
...
+                        {% endif %}
                     </a>
+                    {% if next_article.subtitle %}
+                        {% if theme.slug == 'springster' or theme.slug == 'tujibebe' or theme.slug == 'zathu'  %}
+                            <p class="teaser-stacklist__subtitle"> {{ next_article.subtitle|truncatewords:20 }}</p>
+                        {% else %}
+                            <p class="teaser-stacklist__subtitle"> {{ next_article.subtitle }}</p>
+                        {% endif %}
+                    {% else %}
+                        <p class="teaser-stacklist__subtitle"> - </p>
```
Comment (codiebeulaine) [inline articles/templates/articles/_patterns/article-teasers-recommendations.html:36]: why is the subtitle a dash?

File: articles/templates/articles/_patterns/article-teasers.html [hunk 2]
```diff
@@ -26,14 +26,23 @@ <h4 class="teaser-stacklist-parent">
...
             {% if post.subtitle %}
-                <p class="teaser-stacklist__subtitle"> {{ post.subtitle }}</p>
+                {% if theme.slug == 'springster' or theme.slug == 'tujibebe' or theme.slug == 'zathu'  %}
+                    <p class="teaser-stacklist__subtitle"> {{ post.subtitle|truncatewords:20 }}</p>
+                {% else %}
+                    <p class="teaser-stacklist__subtitle"> {{ post.subtitle }}</p>
+                {% endif %}
             {% else %}
-                <p class="teaser-stacklist__subtitle"> ... </p>
+                <p class="teaser-stacklist__subtitle"> - </p>
```
Comment (codiebeulaine) [inline articles/templates/articles/_patterns/article-teasers.html:43]: same here?

File: home/templates/home/springster/home.html [hunk 3]
```diff
@@ -1,21 +1,36 @@
-{% load comments wagtailcore_tags i18n %}
+{% load comments i18n  %}
```
Comment (codiebeulaine) [inline home/templates/home/springster/home.html:1]: extra space here

File: articles/templates/articles/_patterns/article-teasers.html (hunk 2 above)
Comment (Mitso) [reply articles/templates/articles/_patterns/article-teasers.html:43]: If the subtitle is not included show the dash.

**Comment by Mitso** [comment]
Besides the slashes, I will create a new PR to address those.

**Pull request merged**